lib/autokey/gtkui/configwindow.py

```diff
@@ -868,18 +871,17 @@ def set_dirty(self, dirty):
         self.uiManager.get_action("/MenuBar/File/revert").set_sensitive(dirty)
 
     def config_modified(self):
-        if not self.__warnedOfChanges:
-            Gdk.threads_enter()
-            msg = _("Changes made in other programs will not be displayed until you\
-close and reopen the AutoKey window.\nThis message is only shown once per session.")
-            dlg = Gtk.MessageDialog(self.ui, type=Gtk.MessageType.QUESTION, buttons=Gtk.ButtonsType.OK,
-                                   message_format= _("Configuration has been changed on disk."))
-            dlg.format_secondary_text(msg)
-
-            dlg.run()
-            dlg.destroy()
-            Gdk.threads_leave()
-            self.__warnedOfChanges = True
+        #save tree view selection
```

Comment:
Probably worth putting a logging message in here. Probably at INFO level, eg `"Modifications detected to open files. Reloading..."`